README.md

```diff
@@ -69,13 +67,40 @@ class MyPage(Page):
 
 And render the content in a template:
 
-```
+```html+django
 {% load wagtailmarkdown %}
 <article>
 {{ self.body|markdown }}
 </article>
 ```
 
+You can pass extensions by name to the filter:
+
+```html+django
+{% load wagtailmarkdown %}
+<article>
+{{ self.body|markdown:"my_extension_package.my_extension_module" }}
```

Comment:
Would it not be better to move the extension definition to `MarkdownField` / `MarkdownBlock` rather than doing it in the template?

The rest looks good to me